docs/v6-migration.md

```diff
@@ -259,6 +259,22 @@ _"Missing"_ status is valid.
 **We do not query any external metadata servers.**
 :::
 
+#### Sonarr TV Library Searching
+
+Previously you may have noticed that [data-matching](./tutorials/data-based-matching.md) a Sonarr TV Library would not perform searches "properly", mainly due to the nested folder structures and lack of risky matching supporting multi-file folders. We have now enhanced the search capabilities to correctly identify series and seasons in Sonarr's organized libraries, and with risky multi-file matching - can match season packs you've previously downloaded from your Sonarr Library data back to torrents for cross-seeding.

```

Comment:
I think this, along with the use case on line 272, could be slimmed down into something like the following:

> It is preferable to feed `cross-seed` original, un-renamed data files when possible. However, `cross-seed` can now match season packs after they've been renamed by Sonarr. This can be useful to find cross-seeds from Usenet-downloaded media.